paladin/tests/mod_test.ts

```diff
@@ -0,0 +1,396 @@
+import { Rhum } from "../../test_deps.ts";
```

Comment:
deno fmt this file

Reply:
https://github.com/drashland/deno-drash-middleware/pull/25/commits/be604ab3fce854d077200672fc0fc35d77ab7402